README.md

```diff
@@ -0,0 +1,345 @@
+2. The Codewind backend containers install after you click `Install`. After you click `Install`, the necessary images are downloaded from the internet. The download is approximately 1 GB.
+
+The following images are pulled. These images together form the Codewind backend:
+
+1. `eclipse/codewind-performance-amd64`
+2. `eclipse/codewind-pfe-amd64`
+
+### Configuring Codewind with application stack hubs
+
+Configure Codewind to use application stack hubs so you can focus exclusively on your code:
+
+1. Click the `Codewind` tab. 
+2. Expand `Codewind` by clicking the drop-down arrow.
+3. Right-click `Projects (Local)`.
+4. Select `Manage Template Sources`. 
+5. Select application stack hubs. 
+
+You now have configured Codewind to application stack hubs and can 
+proceed to develop your microservice within Codewind.
+
+### Creating an Appsody project
+
+Appsody helps you develop containerized applications and removes the burden of managing the full software development stack. If you want more context about Appsody, visit the https://appsody.dev/docs[Appsody welcome page]. 
+
+1. Right-click `Projects (Local)` under `Codewind` in the `Codewind` tab.
+2. Select `Create New Project...`
+    * **Note:** Make sure Docker is running. Otherwise, you get an error. 
+3. Name your project `appsody-calculator`. 
+4. Under `Template`, select `Appsody Eclipse MicroProfile template`. 
+    * If you don't see an Appsody template, select the `Manage Template Sources...` link at the end of the window.
+    * Select the `Appsody Stacks - appsodyhub` checkbox. 
+    * Click `OK`.
+    * The templates are refreshed, and the Appsody templates are available. 
+5. Click `Finish`.
+    * To monitor your project's progress, right-click on your project, and select `Show Log Files`.
+    * Select `Show All`. Then a `Console` tab is displayed where you see your project's build logs. 
+
+Your project is displayed in the `Projects (Local)` section. The progress for creating your project is tracked next to the project's name. 
+
+Your project is complete when you see your project is running and its build is successful. 
+
+### Accessing the application endpoint in a browser
+
+1. Return to your project under the Codewind tab. 
+2. Right-click your project and select `Open Application`. 
+
+Your application is now opened in the browser, showing the welcome to your Appsody microservice page. 
+
+### Adding a REST service to your application
+
+1. Go to your project's workspace under the Project Explorer tab. 
+2. Navigate to `Java Resources->src/main/java->dev.appsody.starter`. 
+3. Right-click `dev.appsody.starter` and select `New->Class`.
+4. Create a Class file, name it `Calculator.java`, and select `Finish`. This file is your JAX-RS resource. 
+5. Populate the file with the following code then **save** the file: 
```

Comment:
if you put the code in a separate file, you wouldn't have to display it twice in the article.

Reply:
I'm not quite sure what changes are suggested to be made here?